ext/civigrant/civigrant.php

```diff
@@ -68,26 +68,6 @@ function civigrant_civicrm_permission(&$permissions) {
   ];
 }
 
-/**
- * Implements hook_civicrm_tabSet().
- *
- * Add grants tab to contact summary screen.
- */
-function civigrant_civicrm_tabSet($tabSetName, &$tabs, $context) {
-  if ($tabSetName === 'civicrm/contact/view' && !empty($context['contact_id'])) {
```

Comment:
@eileenmcnaughton I see the permission problem you noted in https://github.com/civicrm/civicrm-core/pull/22064#issuecomment-1009533799 - this was adding the tab unconditionally without checking "access CiviGrant". This PR fixes it by deleting that code :) (the Afform does check that permission)